auth/src/main/java/org/jboss/pnc/bacon/auth/model/CacheFile.java

```diff
@@ -23,7 +23,7 @@
 @Slf4j
 public class CacheFile {
 
-    public static final String CACHE_FILE = Constant.CONFIG_FOLDER + "/" + "saved-user.json";
+    public static final String CACHE_FILE = Config.getConfigLocation() + "/" + "saved-user.json";
```

Comment:
This won't work if the config location is changed after program start - it will still point to the original location.